### PR TITLE
Add org.kde.vvave

### PR DESCRIPTION
**Pull request by ognarb**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. (just a casual contributor to maui projects but the maintainer knows about this MR)
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by ognarb** [comment]
bot, build org.kde.vvave

**Comment by ognarb** [comment]
bot, build org.kde.vvave

**Comment by ognarb** [comment]
bot, build org.kde.vvave

**Comment by ognarb** [comment]
bot, stop org.kde.vvave

**Comment by hfiguiere** [comment]
I stopped it for you. Sorry, the bot can't do that.

**Comment by ognarb** [comment]
> I stopped it for you. Sorry, the bot can't do that.

thanks turn out putting a patch after the source is important.

**Comment by ognarb** [comment]
bot, build org.kde.vvave

File: org.kde.vvave.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "vvave",
+    "tags": ["nightly"],
+    "desktop-file-name-suffix": " (Nightly)",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
```
Comment (hfiguiere) [inline org.kde.vvave.json:13]:
```suggestion
        "--socket=fallback-x11",
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by barthalion** [comment]
The build fails because CMake(?) doesn't add `-lz` to the linker flags correctly.

**Comment by ognarb** [comment]
bot, build org.kde.vvave

**Comment by ognarb** [comment]
bot, build org.kde.vvave

**Comment by ognarb** [comment]
bot, build org.kde.vvave

File: org.kde.vvave.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--own-name=org.mpris.MediaPlayer2.vvave",
+        "--socket=pulseaudio"
+    ],
+    "separate-locales": false,
```
Comment (barthalion) [inline org.kde.vvave.json:18]: Is there some reason not to separate locales?

Comment (ognarb) [reply org.kde.vvave.json]: not really, I just copied the base manifest from another manifest that had this line.

**Comment by nedrichards** [comment]
bot, build org.kde.vvave

**Pull request closed without merging**